recipes/perl-grass/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: perl-grass
+  version: '1.1.6'
+
+build:
+  number: 2
+  skip: True # [osx]
+
+source:
+  fn: v1.1.6.tar.gz
+  url: https://github.com/cancerit/grass/archive/v1.1.6.tar.gz
+  md5: 81acae11f5687d070166cda26f069de3
+
+requirements:
+  build:
+    - perl-threaded
+    - perl-app-cpanminus
+    - perl-file-sharedir-install
+    - perl-module-build
+    - perl-bio-db-sam
+    - perl-bioperl
+    - perl-gd
+    - perl-ipc-system-simple
+    - perl-sanger-cgp-vcf
+    - perl-encode-locale
+    - perl-lwp-simple
+    - perl-xml-parser
+  run:
+    - perl-threaded
+    - perl-ipc-system-simple
+    - perl-bioperl
+    - perl-bio-db-sam
+    - perl-bio-samtools
+    - perl-gd
+    - perl-sanger-cgp-vcf
+
+#test:
+#  commands:
```

Comment:
Yet I'm not sure why `grass.pl` is not finding the `Tabix.pm` module within the bioconda `_test` environment: 

```
[root@node08 perl-grass]# conda build . -c bioconda
(...)
+ grass.pl --help
Can't locate Tabix.pm in @INC (you may need to install the Tabix module) (@INC contains: /anaconda/envs/_test/bin/../lib /anaconda/envs/_test/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi /anaconda/envs/_test/lib/perl5/site_perl/5.22.0 /anaconda/envs/_test/lib/perl5/5.22.0/x86_64-linux-thread-multi /anaconda/envs/_test/lib/perl5/5.22.0 .) at /anaconda/envs/_test/lib/perl5/site_perl/5.22.0/Sanger/CGP/Grass/GenomeData/GenomeDataCache.pm line 32.                    BEGIN failed--compilation aborted at /anaconda/envs/_test/lib/perl5/site_perl/5.22.0/Sanger/CGP/Grass/GenomeData/GenomeDataCache.pm line 32.                                                                                                 Compilation failed in require at /anaconda/envs/_test/lib/perl5/site_perl/5.22.0/Sanger/CGP/Grass/GenomeData.pm line 28.
BEGIN failed--compilation aborted at /anaconda/envs/_test/lib/perl5/site_perl/5.22.0/Sanger/CGP/Grass/GenomeData.pm line 28.
Compilation failed in require at /anaconda/envs/_test/bin/grass.pl line 37.
BEGIN failed--compilation aborted at /anaconda/envs/_test/bin/grass.pl line 37.
TESTS FAILED: perl-grass-1.1.6-2
```

While it seems to be installed already via `perl-bio-db-hts`:

```
[root@node08 perl-grass]# find /anaconda/envs/_test/lib/perl5/ -iname "Tabix.pm"
/anaconda/envs/_test/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi/Bio/DB/HTS/Tabix.pm
```

Oh, while writing this I realized that `Grass/GenomeData/GenomeDataCache.pm` doesn't seem to be packaged in bioconda or even in metacpan... guess I'll have to dig into that first or mangle `PERL5LIB` which I would like to avoid, *sigh*...